### ISSUE TITLE
Permission Denied

### ISSUE THREAD

**Issue by soorajvinodan** [jriall/react-redux-quiz-app#1]
I tried to clone and run your project. But the question json returns permission denied error. Can you please fix it

**Comment by Hamza119612** [comment]
Yeah the same here .. can u allow the use of the json" GET https://quiz-app-bae2b.firebaseio.com/data.json 401 (Unauthorized)"
